### ISSUE TITLE
on blur, terminal textarea is cleared so screen reader user cannot review input

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#29349]
1. with a screen reader on, focus the terminal and type some characters
2. use left and right arrows to inspect the input
3. focus something else and then refocus the terminal and try to do the same
4. 🐛 the textarea has been cleared so it announces blank
